apps/browser/qml/pages/components/PopUpMenuItem.qml

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright (c) 2021 Open Mobile Platform LLC.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import QtQuick 2.2
+import QtQuick.Window 2.2 as QuickWindow
+import QtGraphicalEffects 1.0
+import Sailfish.Silica 1.0
+import Sailfish.Silica.private 1.0 as Private
+import Sailfish.Browser 1.0
+import Sailfish.Policy 1.0
+import Sailfish.WebView.Controls 1.0
+import Sailfish.WebEngine 1.0
+import com.jolla.settings.system 1.0
+import org.nemomobile.dbus 2.0
+import "." as Browser
+import "../../shared" as Shared
+
+Item {
+    id: root
+    property int horizontalOffset
+    property int iconWidth
+    readonly property int verticalPadding: 3 * Theme.paddingSmall
+
+    height: content.height + verticalPadding
```

Comment:
You have a gap of `verticalPadding` at the top, so should put a similar gap at the bottom.

Reply:
Thanks, corrected